server/routes/bassReferral.js

```diff
@@ -18,11 +21,61 @@ module.exports = ({licenceService}) => (router, audited) => {
         res.redirect(`${nextPath}${bookingId}`);
     }));
 
+    router.get('/bassReferral/bassOffer/:bookingId', asyncMiddleware(async (req, res) => {
+
+        const formName = 'bassOffer';
+        const sectionName = 'bassReferral';
+
+        const {bookingId, action} = req.params;
+        const licenceStatus = getLicenceStatus(res.locals.licence);
+
+        const {licenceSection, pageDataMap, validateInPlace} = formConfig[formName];
+        const dataPath = pageDataMap || ['licence', sectionName, licenceSection];
+
+        const data = getIn(res.locals.licence, dataPath) || {};
+        const withdrawnBass = licenceStatus.decisions.bassWithdrawn ? getBassRejections(res.locals.licence).last() : {};
+
+        const errors = firstItem(req.flash('errors')) || {};
+        const pathToErrors = validateInPlace ? [sectionName, formName] : [];
+        const errorObject = getIn(errors, pathToErrors) || {};
```

Comment:
Don't need this anymore. const errorObject = firstItem(req.flash('errors')) || {}; should suffice